src/traces/scatter/index.js

```diff
@@ -458,6 +458,8 @@ scatter.plot = function(gd, plotinfo, cdscatter) {
             line = trace.line;
         if(trace.visible !== true) return;
 
+        d[0].node = this; // store node for tweaking by selectPoints
```

Comment:
Hacky, but much better than doing a complicated `d3.select('path ....')` filter again in `selectPoints`.